arc/arc.go

```diff
@@ -202,6 +202,11 @@ func (c *ARCCache[K, V]) Len() int {
 	return c.t1.Len() + c.t2.Len()
 }
 
+// Cap returns the capacity of cache
+func (c *ARCCache[K, V]) Cap() int {
+	return c.Cap()
```

Comment:
This results in an infinite recursive call.

Reply:
> This results in an infinite recursive call.

Thank for your reviewing, I have fix it